boards/shields/wuxi_ov5640/wuxi_ov5640.overlay

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	chosen {
+		zephyr,camera = &csi;
+	};
+};
+
+&zephyr_cam_i2c {
```

Comment:
This should probably be `nxp_cam_i2c`- the connector used here is specific to NXP

Reply:
Here we name it `zephyr_cam_i2c` with the hope that other boards / vendors can define this name in their own devicetree and reuse the overlay (ov5640 is always attached to an i2c). BTW, the gpio pin number seems something specific to NXP. But in this case, why don't we just use their **real name (as in my first version)** without defining an alias ?

Reply:
Anyways, changed as your suggestion.